treeherder/config/settings.py

```diff
@@ -142,6 +139,17 @@
     'default': env.db_url('DATABASE_URL', default=LOCALHOST_MYSQL_HOST),
 }
 
+SKIP_INGESTION = env('SKIP_INGESTION', default=False)
+
+# Block for changes within the context of local development (venv & Docker)
+if not PRODUCTION:
+    # Changing PULSE_AUTO_DELETE_QUEUES to True when Treeherder is not running in production
+    PULSE_AUTO_DELETE_QUEUES = True
```

Comment:
Moving this in here fixes a different bug where ingestion happening within Docker would create a queue that does not auto-delete. I had been looking for this issue for a while!